tests/kernel/kirk.pm

```diff
@@ -30,11 +34,21 @@ sub run
     my $suite = get_var('KIRK_SUITE', '');
 
     select_serial_terminal;
+    zypper_call("ref");
 
-    zypper_call("in -y git");
-    assert_script_run("git clone -q --single-branch -b $branch --depth 1 $repo");
+    my $cmd = '';
+    if ($install_from =~ /git/i) {
+        is_transactional ? trup_install("git") : zypper_call("in -y git");
+        assert_script_run("git clone -q --single-branch -b $branch --depth 1 $repo");
+
+        $cmd = "python3 kirk/kirk ";
+    } else {
+        add_ltp_repo();
```

Comment:
Shouldn't the repo already be there (from `install_ltp`)? Or is this supposed to be scheduled independently?

Reply:
kirk is contained in the ltp repo yes